src/Core/src/Handlers/Entry/EntryHandler.Windows.cs

```diff
@@ -34,12 +34,12 @@ protected override void ConnectHandler(TextBox platformView)
 		{
 			platformView.KeyUp += OnPlatformKeyUp;
 			platformView.TextChanged += OnPlatformTextChanged;
-			platformView.Loaded += OnPlatformLoaded;
+			platformView.SizeChanged += OnPlatformViewSizeChanged;
```

Comment:
Invalidate the MauiTextBox attached properties when the size changed (loaded, change visibility etc)